frontend/lib/framework/error-classes-list.cpp

```diff
@@ -128,6 +129,68 @@ void ErrorMissingInclude::write(ErrorWriterBase& wr) const {
   wr.note(moduleInclude, "expected file at path '", filePath, "'");
 }
 
+void ErrorUseImportUnknownSym::write(ErrorWriterBase& wr) const {
+  auto visibilityClause = std::get<const uast::VisibilityClause*>(info);
+  auto symbolName = std::get<std::string>(info);
+  auto searchedScope = std::get<const resolution::Scope*>(info);
+  auto useOrImport = std::get<const resolution::VisibilityStmtKind>(info);
+  wr.heading(kind_, type_, locationOnly(visibilityClause),
+             "cannot find symbol '", symbolName, "' for ", useOrImport, ".");
+  wr.message("Symbol named here:");
```

Comment:
Hm, I would say my automatic reaction is to prefer "named", maybe since it emphasizes that a name is what we're trying to resolve. I think this is a good topic for further discussion, since I agree we want to be consistent on this, and this issue comes up on more than just scope-resolve errors I'm sure.

Reply:
My post-hoc justification (after some soul searching) is that "named" is possibly ambiguous; "named" can refer to the act of giving something a name (i.e., I'm naming my new pet). It's plausible to refer to creating a new `const` variable, like `const pi = 3.14`, as "naming" the constant.

Reply:
Oops, didn't mean to ignore this. I'll add on a post-hoc justification to avoid "referenced", which is that `Symbol referenced here:` could potentially be read as meaning "here is (the declaration of) the symbol that was referenced".
With my piece said, I think that's a less likely misunderstanding than the one with "named", so I'll go through this PR and switch  to "referenced".

Reply:
Looks like this is actually moot, because this message has been rephrased to `In the following [use/import] statement:`, and "named" is not currently used in any of the error messages of this PR.